penquins/penquins.py

```diff
@@ -787,16 +807,14 @@ def query_skymap(
         filter = {
             "candidate.jd": {"$gt": jd_start, "$lt": jd_end},
             "candidate.jdstarthist": {
-                "$gt": jd_start,
-                "$lt": jd_end,
-            },
-            "candidate.jdendhist": {
-                "$gt": jd_start,
-                "$lt": jd_end,
+                "$gt": jdstarthist_start,
+                "$lt": jdstarthist_end,
             },
             "candidate.programid": {
                 "$in": program_ids
-            },  # 1 = ZTF Public, 2 = ZTF Public+Partnership, 3 = ZTF Public+Partnership+Caltech
+            },
+            # 1 = ZTF Public, 2 = ZTF Public+Partnership, 3 = ZTF Public+Partnership
```

Comment:
Could you revert that change? Not necessary

Reply:
done